pkgs/top-level/all-packages.nix

```
@@ -28274,6 +28274,8 @@ with pkgs;
 
   swaynag-battery = callPackage ../applications/misc/swaynag-battery {};
 
+  swayest-workstyle = callPackage ../applications/window-managers/sway/swayest-workstyle.nix { };
```

Normally we do one directory for each package or very close related packages, not for a package family. 
```suggestion
  swayest-workstyle = callPackage ../applications/window-managers/sway/swayest-workstyle { };
```

---

Fixed, rebased to master and re-tested in latest rev. Thanks much.

---

Worth noting [there's other packages to move](https://i.ckie.dev/sLTf8tM.txt) if we wanna be consistent.